src/pages/Authentication/Login.jsx

```diff
@@ -1,10 +1,9 @@
 
```

Comment:
Code can be formatted better if you use extension like Prettier or Beautify to allow easy viewing